content/03kapitel.tex

```diff
@@ -2,6 +2,51 @@
 
 \chapter{Roadmap}
 
-\section{Epic 1}
+\section{Überblick}
+Nachdem die Requirements, Rollenaufteilung und Organisation des Projektes geklärt sind, geht es darum zu klären, was später in den tatsächlichen Sprints geschehen soll.
+Als Orientierungshilfe werden hierbei die Requirements des Projektes geordnet und in Product Increments aufgeteilt.
 
-\section{Epic 2}
+In diesem Kontext können \ac{PI}s als die geplanten größeren Versionssprünge unserer Plattform verstanden werden.
+Wichtig ist es hierbei hervorzuheben, dass \ac{PI}s nicht die einzelnen Sprintziele darstellen.
+Diese im Voraus (mit festen zeitlichen Markern) festzulegen hätte wenig mit der im Team erzielten agilen Projektorganisation zu tun.
+Vielmehr stellen die \ac{PI}s des Projektes den geplanten Pfad dar, in welchem die Requirements abgearbeitet werden sollen.
+
+\section{Product Increments}
+Aus der obig erläuterten Überlegung leitet sich sie in Abbildung \ref{fig:roadmap} dargestellte Roadmap ab.
+Die verschiedenen \ac{PI}s sollen nun in den folgenden Abschnitten erläutert werden.
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.3\linewidth]{roadmap.png}
+    \caption{Roadmap des Projektes}
+    \label{fig:roadmap}
+\end{figure}
+
+\subsection{Initial Infrastructure}
+Ziel des \ac{PI}s ``Initial Infrastructure''\footnote{dt. Anfängliche Infrastruktur} ist es die Grundstruktur für die Plattform aufzusetzen und Workflow Elemente wie eine \ac{CI/CD} Pipeline, das Kubernetes Cluster und die initiale Servicestruktur zu erstellen.
+Fragen rund um, wie die Service-Architektur und Infrastruktur aufzusetzen ist, gilt hierbei zu klären.
+Am Ende dieses \ac{PI}s soll das Ergebnis eine funktionsfähige Infrastruktur sein, auf welcher in kommenden Sprints entwickelt und Sprint-Ergebnisse ausgeliefert werden können.
+
+\subsection{Article Views and Base Homepage}
+Ziel des \ac{PI}s ``Article Views and Base Homepage''\footnote{dt. Homepage und Einsichtseiten} ist es die wichtigsten Seiten der Applikation zu erstellen.
+Auf Basis von Mockdaten sollen hierbei die Requirements in Richtung der Einsichtseiten von Feed und Artikel Einsichtseiten bzw. der Homepage mit vorgeschlagenen Artikeln erfüllt werden.
+
+
+\subsection{User Interaction}
+Ziel des \ac{PI}s ``User Interaction''\footnote{dt. Bernutzerinteraktion} soll das Ausbauen des User-Management Requirements sein.
+Hierbei sollen sich Benutzer in die Plattform einloggen können, Feeds zur Plattform hinzufügen und abonnieren können bzw. Artikel liken und speichern können.
+Da es nun einen Weg gibt Artikel zur Plattform hinzuzufügen, ist es ebenfalls Teil dieses \ac{PI}s statt Mockdaten echte RSS Feeds zu verwenden.
+
+\subsection{Recommendations}
+Ziel des \ac{PI}s ``Recommendations''\footnote{dt. Vorschläge} ist es die Recommendations der Plattform auszubauen.
+Requirements wie das Angeben von Präferenzen in Richtung Kategorien und vorgeschlagenen Feeds gilt es hier zu implementieren.
```

Comment:
```suggestion
Anforderungen wie das Angeben von Präferenzen mithiolfe von Kategorien und vorgeschlagenen Feeds gilt es hier zu implementieren.
```

Reply:
fixed typeo in your suggestion and added it